server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/JmapCommonRequests.java

```diff
@@ -131,6 +131,46 @@ public static String getLastMessageId(AccessToken accessToken) {
                 .path(ARGUMENTS + ".messageIds[0]");
     }
 
+    public static String getLatestMessageId(AccessToken accessToken, Role mailbox) {
+        String inboxId = getMailboxId(accessToken, mailbox);
+        return with()
+                .header("Authorization", accessToken.serialize())
+                .body("[[\"getMessageList\", {\"filter\":{\"inMailboxes\":[\"" + inboxId + "\"]}}, \"#0\"]]")
```

Comment:
what about adding a sort by date in this request, to ensure to not break this method in case of changing the default sort order?